modules/network/kademlia/kademlia.js

```diff
@@ -639,161 +641,59 @@ class Kademlia {
                 });
             };
 
-            node.sendDataLocationResponse = async (message, contactId) => {
-                const { contact, header } = await node.getContact(contactId);
-                return new Promise((resolve, reject) => {
-                    node.send('kad-data-location-response', { message, header }, contact, (err, res) => {
-                        if (err) {
-                            reject(err);
-                        } else {
-                            resolve(res);
-                        }
-                    });
-                });
-            };
+            node.replicationRequest = async (message, contactId) =>
+                node.sendDirectMessage(message, contactId, 'kad-replication-request');
 
-            node.dataReadRequest = async (message, contactId) => {
-                const { contact, header } = await node.getContact(contactId);
-                return new Promise((resolve, reject) => {
-                    node.send('kad-data-read-request', { message, header }, contact, (err, res) => {
-                        if (err) {
-                            reject(err);
-                        } else {
-                            resolve(res);
-                        }
-                    });
-                });
-            };
+            node.replacementReplicationRequest = async (message, contactId) =>
+                node.sendDirectMessage(message, contactId, 'kad-replacement-replication-request');
```

Comment:
Can we do something like this. We put all methods and routs in array:
const requestNameArray = [{methodName: 'replacementReplicationRequest', routeName: 'kad-replacement-replication-request}, ....];

and than

requestNameArray.forEach((element)=>{
      node[element.methodName]= async (message, contactId) => 
               node.sendDirectMessage(message, contactId, element.routeName);
})

That way when we want to add new route we just need to add new element to array, and we can remove this method definition from kademlia class